.env.example

```
@@ -22,22 +22,23 @@ JWT_AUDIENCE=Web
 EMAILSETTINGS__PORT=587
 EMAILSETTINGS__FROM=
 EMAILSETTINGS__USERNAME=
-EMAILSETTINGS__PASSWORD=""
+EMAILSETTINGS__PASSWORD=
 
+# Cloudinary
 CLOUDINARY__CLOUDNAME=
 CLOUDINARY__APIKEY=
-CLOUDINARY__APISECRET=
-
+CLOUDINARY__APISECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)